custom/icds_reports/migrations/0188_add_child_beneficiary_status_fields_to_chm.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-04-17 09:26
+from __future__ import unicode_literals
+
+
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from custom.icds_reports.utils.migrations import get_view_migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('icds_reports', '0187_chm_view_growth_tracker'),
+    ]
+
+    operations = [
+        migrations.RunSQL("ALTER TABLE child_health_monthly ADD COLUMN valid_in_month_end smallint"),
```

Comment:
May be good to call the columns as <xyz>_status_daily, something like valid_status_daily, migration_status_daily. So that it can be easily distinguishable from the monthly only.

Reply:
changed